test-cypress/integration/import/onto-refine.spec.js

```diff
@@ -0,0 +1,132 @@
+describe('Import/ OntoRefine', () => {
+
+    let repositoryId;
+
+    beforeEach(() => {
+        repositoryId = 'onto-refine-' + Date.now();
+        cy.createRepository({id: repositoryId});
+
+        cy.visit('/ontorefine');
+
+        cy.get('.ot-splash').should('not.be.visible');
+        getOntoRefineFrame().should('be.visible');
+    });
+
+    afterEach(() => {
+        cy.deleteRepository(repositoryId);
+    });
+
+    const REFINE_TABS = ['Create Project', 'Open Project', 'Import Project', 'Language Settings'];
+    const CREATE_PROJECT_TABS = ['This Computer', 'Web Addresses (URLs)', 'Clipboard'];
+
+    it('should properly render OntoRefine transformation tool frame', () => {
+
+        // Refine should provide navigation tabs
+        getOntoRefineTabs()
+            .should('be.visible')
+            .and('have.length', 4)
+            .each(($tab, $index) => expect($tab.text()).to.equal(REFINE_TABS[$index]))
+            .eq(0)
+            // The first tab should be automatically selected
```

Comment:
and visible maybe?

Reply:
Add explicit check that the default tab is visible. The others should be handled by `.click()`'s default checks for visibility.